src/strategies/staked-uniswap/UniV2.json

```diff
@@ -0,0 +1,568 @@
+{
```

Comment:
We don't need the whole file here, it seem that only `balanceOf` and `totalSupply` inputs are needed.

Reply:
Good point, I've removed the file